Source/iop/Iop_Intc.h

```diff
@@ -36,6 +36,8 @@ namespace Iop
 			LINE_DMA_DEV9 = 0x29,
 			LINE_DMA9 = 0x2A,  //sif0
 			LINE_DMA10 = 0x2B, //sif1
+			LINE_DMA11 = 0x2C, // SIO2i

```

Comment:
Let's name these `LINE_DMA_SIO2in` and `LINE_DMA_SIO2out`.

Reply:
Done